vid_sdl2.c

```diff
@@ -562,6 +568,11 @@ static void VID_SDL_Init(void)
 		}
 	}
 
+	SDL_SetHint(SDL_HINT_VIDEO_MAC_FULLSCREEN_SPACES, "0");
```

Comment:
Shouldn't this be inside of the `#ifdef __APPLE__` as well? I guess it is just ignored on other platforms...